pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicy/_http.log

```diff
@@ -29,6 +29,8 @@ X-Xss-Protection: 0
   "selfLink": "https://www.googleapis.com/compute/v1/locations/global/operations/${operationID}",
   "startTime": "2024-04-01T12:34:56.123456Z",
   "status": "RUNNING",
+  "targetId": "${firewallPolicyId}",
+  "targetLink": "https://www.googleapis.com/compute/v1/locations/global/firewallPolicies/${firewallPolicyId}",
```

Comment:
I think targetId and targetLink are necessary in mock compute operation?

Reply:
Not sure, but whatever the real-gcp does, that's what we want to simulate.

Reply:
Update the log to match realGCP